File: Practice(Beginner)/Python3/Chef_and_Dolls.py
# Source: https://www.codechef.com/problems/MISSP

t = int(input())
for test in range(t):
    n = int(input())
    d = {}
    for i in range(n):
        temp = int(input())
        if(temp in d):
            d[temp] += 1
        else:
            d[temp] = 1
    for key, val in d.items():
        if(val%2 != 0):
            print(key)
            break

File: Practice(Beginner)/C++/Chef and Dolls.cpp
// Source: https://www.codechef.com/problems/MISSP

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t, n, temp;
    cin >> t;
    while (t--){
        cin >> n;
        unordered_map<int, int> dolls;
        for (int i = 0; i < n; ++i) {
            cin >> temp;
            dolls[temp]++;
        }
        for(auto node: dolls){
            if(node.second % 2 != 0){
                cout << node.first << endl;
                break;
            }
        }
    }
    return 0;
}